packages/common/src/types/savedCharts.ts

```diff
@@ -583,6 +583,32 @@ export const getChartKind = (
     }
 };
 
+export const getEChartsChartTypeFromChartKind = (
```

Comment:
our `ChartKind` types have a different value from the `Echarts`' ones. I've created a util bc I haven't seen one currently being used.